search/search_repository_blackbox_test.go

```diff
@@ -69,6 +69,82 @@ func (s *searchRepositoryBlackboxTest) getTestFixture() *tf.TestFixture {
 	)
 }
 
+func (s *searchRepositoryBlackboxTest) TestSearchWithChildIterationWorkItems() {
```

Comment:
Can you please add a test for `child:true` and another one for `/** with child:false`?

Reply:
The "child" attribute is not recognized by the parser, and it won't make any difference. So, I am not sure we need to test it.

Reply:
There is no support for `"child": true`. The only supported format is `ID/**`.